packages/Webkul/Sitemap/src/Models/Category.php

```diff
@@ -9,6 +9,13 @@
 
 class Category extends BaseCategory implements Sitemapable
 {
+    protected function rootCategoryIds()
+    {
+        return cache()->remember('root_category_ids', 300, function () {
+            return core()->getAllChannels()->pluck('root_category_id')->toArray();
+        });
+    }
```

Comment:
Just to clarify, if you check the `config/repository.php` file, we have already cached the channel. So, I don’t think we need to add caching here, if I am not mistaken.